application.js

```diff
@@ -8,49 +8,46 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+  var email = "dleuliette@gmail.com";  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
-
-
+  document.getElementById("email").value = "fourny.valentin@gmail.com";
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
+  $('#email-hint').html("<strong>This is my email now</strong>");
 
 
   // Ex 4. Add the .blue CSS class to the table header cells (th elements)
-
-
+  $('th').addClass("blue");
 
   // Ex 5. Count the number of table body rows there are (team count!)
-  var team_count = 0;  // TODO: replace 0, keep the team_count variable.
-
+  var team_count = $('.table > tbody > tr').length;  // TODO: replace 0, keep the team_count variable.
 
 
   // Ex 6. Say there is a 15th team added to the table.
   //       Add a row at the bottom, this new team should have zero points.
-
+  $('.table > tbody').append('<tr><td>'+(team_count+1)+'</td><td>Nouvelle Team</td><td>0</td></tr>');
 
 
   // Ex 7. Write some code to sum all points given to all teams
-  var sum = 0;  // TODO: replace 0 with your sum result, keep the sum variable.
-
+  var sum=0;
+  for(var i=0; i<team_count; i++){
+    sum += parseInt($('td:last-child')[i].innerHTML); // TODO: replace 0 with your sum result, keep the sum variable.
+  } 
 
 
   // Ex 8. Change the background color of all table header cells to #DDF4FF
-
+  $('th').css("background-color", "#DDF4FF");
 
 
   // Ex 9. Translate the #results list 20px downward (animation would be great ^_^)
-
+  $('#results').css("margin-top", "-20px");
```

Comment:
It's great but this solution is better

      $('#results').css({ 'transform': 'translateY(20px)' });